plugins/GSdx/Renderers/OpenGL/glext_extra.h

```diff
@@ -0,0 +1,554 @@
+// FIX compilation issue with older Mesa 10 on older distribution
+// Ubuntu 16.04 requires at least the GL45 and GL46 parts
+// imported from glcorearb.h
```

Comment:
Feels odd for these comments to be above the copyright header instead of below just before the code starts.